### PR TITLE
docs: manual profiles

### PR DESCRIPTION
**Pull request by wgwz**
## Description

Closes: #364 

Markdown Preview: https://github.com/regen-network/regen-server/blob/kyle/364-manual-profiles/docs/manual-profiles.md

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] targeted `dev` branch
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] submit a PR against `master` branch after this one (and other PRs depending on this one, e.g. on regen-network/regen-web, if applicable) get merged

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: docs/manual-profiles.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+1. Creating a new account and profile
+2. Updating an already existing profile
+
+# Creating a new account and profile
+
+## Step 1: creating an account for an address
+
+If you need a user type account:
+
+```
```
Comment (blushi) [inline docs/manual-profiles.md:16]:
```suggestion
```sql
```
probably we should use this everywhere

File: docs/manual-profiles.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                  id
+--------------------------------------
+ 997ea872-4c59-11ee-98a7-c26700a3bae4
+(1 row)
+```
+
+## Step 3: updating the profile for the new account
+
+```sql
+# UPDATE
```
Comment (blushi) [inline docs/manual-profiles.md:47]:
```suggestion
UPDATE
```
Should we drop the leading #? with the `sql` mention it renders this as a comment

File: docs/manual-profiles.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+Regardless of which scenario you are working with, it is best practice to first do the process in the staging environment and then do the process in the production environment.
+After completing these changes you can check the results.
+
+For staging:
+
+- https://dev.app.regen.network/ecocredits/accounts/regen12345/portfolio
+
+For production:
+
+- https://app.regen.network/ecocredits/accounts/regen12345/portfolio
```
Comment (blushi) [inline docs/manual-profiles.md:112]: side note that we might want to update that as part of https://github.com/regen-network/regen-web/issues/2076 (even though we'll set up redirects)

**Review by blushi — COMMENTED** [review]
looks good, just a few formatting issues

File: docs/manual-profiles.md (hunk 3 above)
Comment (wgwz) [reply docs/manual-profiles.md:112]: i thought we still planned to keep the address version of this? but we plan to also support a UUID there?

Comment (ryanchristo) [reply docs/manual-profiles.md:110]: The url will change though, see use of `/profile` in the issue.

Comment (wgwz) [reply docs/manual-profiles.md:110]: oh i see thanks for clarifying that. i guess we should put the new version here, since that will be implemented soon

Comment (wgwz) [reply docs/manual-profiles.md]: updated from:
https://app.regen.network/ecocredits/accounts/regen12345/portfolio
to:
https://app.regen.network/ecocredits/profiles/regen12345

Comment (blushi) [reply docs/manual-profiles.md]: this should just be https://app.regen.network/profiles/regen12345 (without "ecocredits")

Comment (wgwz) [reply docs/manual-profiles.md]: whoops!

**Review by blushi — APPROVED** [review]

**Pull request merged**